config/crd/bases/osp-director.openstack.org_openstackconfiggenerators.yaml

```diff
@@ -46,6 +46,13 @@ spec:
             description: OpenStackConfigGeneratorSpec defines the desired state of
               OpenStackConfigGenerator
             properties:
+              enableFencing:
+                description: 'EnableFencing allows the automatic creation of required
+                  heat environment files to enable fencing. Note: - Production OSP
```

Comment:
should we default to true since it's required for prod?

Reply:
we could, I thought it might also be good that this is a must set parameter either true/false as it also requires to install fence-agents-kubevirt.

Reply:
ok, makes sense